js_modules/dagit/packages/core/src/assets/__tests__/AssetPartitions.test.tsx

```diff
@@ -194,11 +195,29 @@ describe('AssetPartitions', () => {
     });
 
     await userEvent.click(screen.getByTestId('sort-1'));
+    await waitFor(async () => {
+      await userEvent.click(screen.getByTestId('sort-reverse-original'));
+    });
     await waitFor(() => {
       expect(
         getByTestId(screen.getByTestId('partitions-zstate'), 'asset-partition-row-WV-index-0'),
       ).toBeVisible();
     });
+
+    await userEvent.click(screen.getByTestId('sort-1'));
+    await waitFor(async () => {
```

Comment:
Need to use waitFor because of the closing animation on popover. For a bit there will be two menus open so just wait until only one of them is open (the one we're opening).